<robot name="peg_0001">
  <link name="peg_0001">
    <inertial>
      <origin xyz="0.0 0.0 -0.085" rpy="-1.570759 0 0"/>
      <mass value="0.050" />
      <inertia  ixx="0.0004849452" ixy="0.0"  ixz="0.0"  iyy="0.0000386667"  iyz="0.0"  izz="0.0004849452" />
    </inertial>
    <visual>
      <origin xyz="0.0 0.0 0.0" rpy="-1.570759 0 0"/>
	<geometry>
		<mesh filename="package://sawyer_sim_examples/models/peg_hole/STL/rigid_peg_0001.STL" scale="0.001 0.001 0.001"/>
	</geometry>
    </visual>
    <collision>
      <origin xyz="0.0 0.0 -0.085" rpy="0 0 0"/>
	<geometry>
		<cylinder length="0.170" radius="0.02"/>
	</geometry>
    </collision>
  </link>
  <gazebo reference="peg_0001">
    <material>Gazebo/White</material>
     <mu1>10000.0</mu1>
     <mu2>10000.0</mu2>
  </gazebo>
</robot>
